src/cursor-doc/paredit.ts

```diff
@@ -792,6 +792,64 @@ export async function close(
   }
 }
 
+function onlyWhitespaceOnSameLineAsCursor(cursor: LispTokenCursor) {
+  cursor = cursor.clone();
+  while (!cursor.atStart()) {
+    switch (cursor.getPrevToken().type) {
+      case 'eol':
+        return true;
+      case 'ws':
+        cursor.previous();
+        continue;
+      default:
+        return false;
+    }
+  }
+}
```

Comment:
This checks if there is only whitespace before the cursor, right? Thinking that the name says something else.

Also, I think we collect all consecutive whitespace on a line as a single token. Maybe that means that there is only whitespace before the cursor if the `getPrevToken().type === 'ws' && getPrevToken().offset === 0`? (Not sure about this.)

Reply:
You are right, fixed here: https://github.com/isaksky/calva/commit/4457629ee446cba9e0da9cef5e09d48c9e7b81f1